jwt/jwt.go

```diff
@@ -0,0 +1,37 @@
+package jwt
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+var secretKey = []byte("farawin")
+
+func CreateToken(id string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"id":  id,
+		"exp": time.Now().Add(time.Hour * 24).Unix(),
```

Comment:
using explicit naming doesn't hurt anybody

Reply:
explicit naming for "CreateToken"?

Reply:
No, in line 15, the key name is exp.
Find a better name for that